core/templates/domain/question/question-backend-api.service.ts

```diff
@@ -181,7 +181,7 @@ export class QuestionBackendApiService {
    * Returns a list of questions based on the list of skill ids and number
    * of questions requested.
    */
-  fetchQuestions(
+  fetchQuestionsAsync(
```

Comment:
Still missing async keyword here and before fetchQuestionSummariesAsync

Reply:
I have updated